xmlbeans/src/test/java/org/n52/svalbard/encode/InsertSensorRequestEncoderTest.java

```diff
@@ -93,47 +93,77 @@ public void prepare() {
     }
 
     @Test
-    public void shoudlThrowExceptionWhenNullValueReceived() throws EncodingException {
+    public void shouldThrowExceptionWhenNullValueReceived() throws EncodingException {
         thrown.expect(UnsupportedEncoderInputException.class);
-        thrown.expectMessage("null input received.");
-        new InsertSensorV2RequestEncoder().create(null);
+        thrown.expectMessage(Is.is("Encoder " +
+                InsertSensorRequestEncoder.class.getSimpleName() +
+                " can not encode 'null'"));
+        encoder.create(null);
     }
 
     @Test
-    public void shoudlThrowExceptionWhenProcedureDescriptionFormatIsMissing() throws EncodingException {
+    public void shouldThrowExceptionIfServiceIsMissing() throws EncodingException {
         thrown.expect(UnsupportedEncoderInputException.class);
-        thrown.expectMessage("procedure description format not defined in InsertSensorRequest.");
-        new InsertSensorV2RequestEncoder().create(new InsertSensorRequest());
+        thrown.expectMessage(Is.is("Encoder " +
```

Comment:
Is it `thrown.expectMessage(Is.is(text))` more readable than `thrown.expectMessage(text)`?

Reply:
Used static imports.